paddle/fluid/pybind/global_value_getter_setter.cc

```diff
@@ -144,44 +242,102 @@ void BindGlobalValueGetterSetter(pybind11::module *module) {
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_check_nan_inf, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_cpu_deterministic, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_enable_rpc_profiler, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_multiple_of_cupti_buffer_size,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_reader_queue_speed_test_mode,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_enable_parallel_graph, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_pe_profile_fname, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_print_sub_graph_dir, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_fraction_of_cpu_memory_to_use,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_fuse_parameter_groups_size,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_fuse_parameter_memory_size,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_init_allocated_mem, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_initial_cpu_memory_in_mb,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(
+      FLAGS_memory_fraction_of_eager_deletion, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_use_pinned_memory, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_benchmark, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_inner_op_parallelism, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_tracer_profile_fname, true);
 #ifdef PADDLE_WITH_CUDA
-  REGISTER_GLOBAL_VAR_GETTER_SETTER(FLAGS_gpu_memory_limit_mb);
-  REGISTER_GLOBAL_VAR_GETTER_SETTER(FLAGS_cudnn_deterministic);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_gpu_memory_limit_mb, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_cudnn_deterministic, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_conv_workspace_size_limit,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(
+      FLAGS_cudnn_batchnorm_spatial_persistent, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_cudnn_exhaustive_search, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_eager_delete_scope, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_fast_eager_deletion_mode,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(
+      FLAGS_fraction_of_cuda_pinned_memory_to_use, true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_fraction_of_gpu_memory_to_use,
+                                           true);
+  REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_initial_gpu_memory_in_mb,
+                                           true);
+  // REGISTER_PUBLIC_GLOBAL_VAR_GETTER_SETTER(FLAGS_reallocate_gpu_memory_in_mb,
```

Comment:
Change these flags as I commented at the head of this file

Reply:
Done.